File: backend/Domain/Entities/Service.cs
using System.ComponentModel.DataAnnotations.Schema;
using System.Xml.Linq;

namespace Domain.Entities
{
    public class Service : BaseLogDomain, IBaseDomain
    {
        public int Id { get; set; }
        public string? Name { get; set; }
        public string? ShortName { get; set; }
        public string? Code { get; set; }
        public string? Description { get; set; }
        public int? DayCount { get; set; }
        public int? WorkflowId { get; set; }
        public decimal? Price { get; set; }
        public string? WorkflowName { get; set; }
        public bool? IsActive { get; set; }
    }
}

File: frontend/src/constants/Service.ts
export type Service = {
  id: number;
  name: string;
  short_name: string;
  code: string;
  description: string;
  day_count: number;
  workflow_id: number;
  workflow_name?: string;
  price: number;
};


export type ServiceCreateModel = {
  id: number;
  name: string;
  short_name: string;
  code: string;
  description: string;
  day_count: number;
  workflow_id: number;
  price: number;
};
